front-end/src/components/Admin/Record/RecordEditor/index.js

```diff
@@ -99,6 +104,10 @@ class RecordEditor extends Component {
         this.props.history.push({
           pathname: '/admin/' + tabKey + '/' + editorUrl
         });
+
+        for (let i = 0; i < formInputs.length; i++) {
+          resizeInput(formInputs[i]);
+        }
```

Comment:
It would be nice, if you could change the resizeInput function, so that you can pass formInputs array to it:
resizeInput(formInputs);
and it would resize all inputs from it. So that you do wouldn't need to write the cycle every time.
You can do this in a separate PR, if you wish.